lib/ansible/modules/cloud/amazon/_ec2_ami_search.py

```diff
@@ -16,7 +16,9 @@
 ---
 module: ec2_ami_search
 short_description: Retrieve AWS AMI information for a given operating system.
-deprecated: "Use M(ec2_ami_find) instead."
+deprecated:
+  version: "2.4"
```

Comment:
This goes back a lot further.  I believe it's 2.0: https://github.com/ansible/ansible-modules-core/tree/stable-2.0/cloud/amazon

Reply:
Yup, so I think this should be deleted